requirements-integration.in

```diff
@@ -4,8 +4,8 @@ aiohttp
 jsonschema<4.18
 # Pinning to <4.0 due to compatibility with the 3.1 controller version
 juju<4.0
-# Need this version of kfp to work with kfp 2.0.0-alpha.7
-kfp==1.8.22
+# Install the latest supported version of KFP SDK
+kfp<3.0
```

Comment:
Do we need `kfp` here at all?  I think this is only needed when trying to update the python examples (eg: whenever a developer is running `tests/integration/pipelines/*.py`).  Can't we just remove it entirely, and note which version is needed in the `*.py` files?  Maybe by providing a requirements.txt for each example `.py`.

Since there are examples for both sdk v1 and v2, no single version of kfp applies to all tests here.

Reply:
Unfortunately we need kfp. We use it to create a client to talk to the `apiserver` when we upload pipelines, create runs, experiments, etc.

Reply:
ohhh right like [here](https://github.com/canonical/kfp-operators/blob/240adae542544b35194e86e18715dfa601c1bc5e/tests/integration/conftest.py#L80).  Ok makes sense